### DISCUSSION TITLE
Configuring the App After Hiding the Icon

### DISCUSSION THREAD

**Discussion by JosephLeedy** [rxhanson/Middle-Localization#47] [Q&A] [answered]
Once you choose to hide the menu bar icon, how do you configure the app again? I'd like to tweak some of the settings, including having it run on login.

**Comment by rxhanson** [comment, marked as answer]
Relaunch the app from Finder, and the prefs window should appear.

**Reply by JosephLeedy** [reply]
Ah, that's so simple that I feel like an idiot for not even having thought of it.

**To clarify the answer for future readers:** Open Finder, go to the "Applications" folder and double-click on Middle.app.
